tests/kafkatest/services/security/security_config.py

```diff
@@ -73,7 +97,9 @@ class SecurityConfig(TemplateRenderer):
     KRB5CONF_PATH = "/mnt/security/krb5.conf"
     KEYTAB_PATH = "/mnt/security/keytab"
 
-    ssl_stores = Keytool.generate_keystore_truststore('.')
+    ssl_stores = SslStores()
```

Comment:
@ewencp Thank you for the review. SSL configuration files are static since all clients and Kafka instances corresponding to a test need to access the same set of files (trust chain in particular). That hasn't changed in this PR. There is one new test which modifies `SecurityConfig.ssl_stores` in order to create certificates with invalid host name. But that also relies on the static value to ensure that the same set of stores is used by Kafka and all clients (ie. the trust chain is valid, only the hostname is invalid). The alternative would be to create a single `ssl_store` that is passed around as parameter to every object that uses security, but that feels like a large pervasive change just to handle a single test that needs to modify the store.